Project.toml

```diff
@@ -5,6 +5,7 @@ repository = "https://github.com/microsoft/OpticSim.jl"
 version = "0.5.1"
 
 [deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
```

Comment:
I don't think BenchmarkTools should be a dependency of the package itself - it's already [installed separately for tests](https://github.com/microsoft/OpticSim.jl/blob/9eec4c489c958847c55c83c01c559b614ab87b64/test/Project.toml#L2)

Reply:
See comment below.